### PR TITLE
Add support for GHC-8.10

### PR DESCRIPTION
**Pull request by mbj**

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -10,27 +10,24 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ghc: ['8.8']
+        ghc: ['8.8', '8.10']
         os: [ubuntu-latest, macOS-latest]
     steps:
     - uses: actions/checkout@v2
-    - uses: actions/setup-haskell@v1.1
+    - uses: actions/setup-haskell@v1
```
Comment (snusnu) [inline .github/workflows/ci.yaml:17]: @mbj I'm curious, why the downgrade?

Comment (mbj) [reply .github/workflows/ci.yaml:17]: @snusnu its actually an upgrade, this tag is supposed to point at the latest `v1` release, see: https://github.com/actions/setup-haskell/tags

Comment (snusnu) [reply .github/workflows/ci.yaml:17]: @mbj Ah, sorry for the noise then. I should've been able to find that out myself.

Comment (mbj) [reply .github/workflows/ci.yaml:17]: @snusnu In fact `setup-haskell` used to slip on updating the "sliding tag". They just recently stepped up their game.

**Pull request merged**